src/DeveloperEmployee.java

```diff
@@ -0,0 +1,31 @@
+import java.util.Scanner;
+
+public class DeveloperEmployee extends Employee {
+	private int devGeneralLevel;
```

Comment:
add some validation here (i.e. from 1 to 10)

Reply:
I added the validation for this attribute.